src/EventSourcingTests/Projections/ViewProjections/simple_multi_stream_projection_with_one_to_many.cs

```diff
@@ -125,6 +125,7 @@ public simple_multi_stream_projection_with_one_to_many()
         {
             StoreOptions(_ =>
             {
+                _.DatabaseSchemaName = "simple_multi_stream_projection_one_to_many";
```

Comment:
The test harness automatically sets the schema name to the class name of the test fixture